tools/python/boutiques/nexusHelper.py

```diff
@@ -0,0 +1,155 @@
+        self.sandbox = sandbox
+        self.no_int = no_int
+        self.verbose = verbose
+        self.config_file = self.get_config_file()
+        self.nexus_endpoint = self.get_nexus_endpoint()
+
+    def verify_nexus_input(self, input_token, input_org, input_project):
+        access_token = input_token
+        org = input_org
+        project = input_project
+        if access_token is None:
+            access_token = self.get_nexus_access_token()
+        if org is None:
+            org = self.get_nexus_organization()
+        if project is None:
+            project = self.get_nexus_project()
+        self.nexus_test_api(access_token, org, project)
+        self.save_nexus_inputs(access_token, org, project)
+        return access_token, org, project
+
+    def get_nexus_access_token(self):
+        json_creds = self.read_credentials()
+        if json_creds.get(self.config_token_property_name()):
+            return json_creds.get(self.config_token_property_name())
+        if self.no_int:
+            raise_error(NexusError, "Cannot find Nexus credentials.")
+        prompt = ("Please enter your Nexus access token (it will be "
+                  "saved in {0} for future use): ".format(self.config_file))
+        self.prompt(prompt)
+
+    def get_nexus_organization(self):
+        json_creds = self.read_credentials()
+        if json_creds.get("nexus-organization"):
+            return json_creds.get("nexus-organization")
+        if self.no_int:
+            raise_error(NexusError, "Cannot find Nexus organization.")
+        prompt = ("Please enter the Nexus organization you want to publish to"
+                  " (it will be saved in {0} for future use): "
+                  .format(self.config_file))
+        self.prompt(prompt)
+
+    def get_nexus_project(self):
+        json_creds = self.read_credentials()
+        if json_creds.get("nexus-project"):
+            return json_creds.get("nexus-project")
+        if self.no_int:
+            raise_error(NexusError, "Cannot find Nexus project.")
+        prompt = ("Please enter the Nexus project you want to publish to"
+                  " (it will be saved in {0} for future use): "
+                  .format(self.config_file))
+        self.prompt(prompt)
+
+    def save_nexus_inputs(self, access_token, org, project):
+        json_creds = self.read_credentials()
+        json_creds[self.config_token_property_name()] = access_token
+        json_creds["nexus-organization"] = org
+        json_creds["nexus-project"] = project
+        with open(self.config_file, 'w') as f:
+            f.write(json.dumps(json_creds, indent=4, sort_keys=True))
+        if self.verbose:
+            print_info("Nexus access token, organization and project"
+                       " saved in {0}".format(self.config_file))
+
+    def get_nexus_endpoint(self):
+        # change once nexus is setup
+        endpoint = "https://sandbox.bluebrainnexus.io/v1" if self.sandbox \
```

Comment:
Could you mention this URL in the helptext so that users know where to create an account?